metrics/platform-metrics-interceptor/src/main/java/ddf/metrics/interceptor/AbstractMetricsInterceptor.java

```diff
@@ -43,20 +46,18 @@
     private static final JmxReporter reporter = JmxReporter.forRegistry(metrics)
             .inDomain(REGISTRY_NAME).build();
 
-    final Histogram messageLatency;
+    static final Histogram messageLatency = metrics.register(MetricRegistry.name(HISTOGRAM_NAME), new Histogram(new SlidingTimeWindowReservoir(1, TimeUnit.MINUTES)));
```

Comment:
Static final vars should be all caps. I'm surprised our checkstyle thing didn't pick it up.